modules/programs/alot-accounts.nix

```diff
@@ -13,6 +13,29 @@ with lib;
       '';
     };
 
+    contactCompletion = mkOption {
+      type = types.attrsOf types.str;
```

Comment:
This needs to be
```
type = with types; nullOr (attrsOf str);
```
otherwise it would never be possible to assign `null` to the option.

Reply:
It's not a problem to have an empty section so instead I always write [[[abook]]] section. No need to check for null so it simplifies things.